src/machines/auth-token-polling-machine.ts

```diff
@@ -108,5 +108,8 @@ export const authTokenPollingMachine = createMachine<Context, Event>(
         return typeof context.authToken !== 'string'
       },
     },
+    delays: {
+      WAIT_BETWEEN_POLLS: 2000,
+    },
```

Comment:
This does two things:
- gives a name to a magic number which provides some additional context
- makes the delay configurable by the interpreter of the machine

The latter gives us more flexibility in a testing context.